app/data.py

```diff
@@ -43,7 +43,7 @@ def projection(self, collection: str, query: Dict, projection: Dict) -> List[Dic
         projection["_id"] = False
         return list(self.collection(collection).find(query, projection))
 
-    def update(self, collection: str, query: Dict, update_data: Dict) -> Tuple:
```

Comment:
Good Job implementing a boolean over a tuple. Removing unnecessary tuples is always suitable for performance and is a part of best practice.